internal/app/tfsec/rules/aws/elasticache/encrypt_replication_group_rule.go

```diff
@@ -1,4 +1,4 @@
-package elasticsearch
+package elasticache
```

Comment:
You need to update `Service: ` at line 20. Can't suggest it as it's not part of the PR :-(